lua/ulx/modules/sh/cfc_fps.lua

```diff
@@ -126,52 +128,79 @@ if SERVER then
         net.Send( targetPlys )
 
         timer.Simple( duration + wiggleRoom, function()
-            cmd.assembleAndYapFPSData( caller, targetPlys )
+            cmd.assembleAndYapFPSData( caller, duration, targetPlys )
         end )
     end
 
     function cmd.checkFPS( caller, targetPlys, duration )
+        local cur = CurTime()
```

Comment:
You localized `CurTime` here, use it through the rest of the function

Reply:
👍